src/DIRAC/WorkloadManagementSystem/Service/PilotManagerHandler.py

```diff
@@ -74,23 +74,12 @@ def export_getCurrentPilotCounters(cls, attrDict={}):
         return S_OK(resultDict)
 
     ##########################################################################################
-    types_addPilotTQReference = [list, int, str, str]
+    types_addPilotReferences = [list, str]
 
-    @deprecated("Use addPilotTQRef")
```

Comment:
This deprecation was introduced for v9.

Reply:
Ah yes indeed.
How should we handle that?

Above you said:
> The name of the method should then be changed to export_addPilotReference, and taskQueueID would not be needed anymore. But really it should be better to then deprecate the old method name (could be done in v8r0).

Should I make a PR targeting v8.0 to introduce `addPilotReferences` and deprecate `addPilotTQReference` (which would be removed from v9.0)?

Reply:
The deprecation was introduced in https://github.com/DIRACGrid/DIRAC/pull/7157 because of the DB simplifications and remove of DN. Maybe what you are suggesting would work.

Reply:
I created https://github.com/DIRACGrid/DIRAC/pull/7358